src/objective-c/tests/CFStreamTests/CFStreamClientTests.mm

```diff
@@ -0,0 +1,208 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#import <XCTest/XCTest.h>
```

Comment:
ideally we avoids forking a copy of existing test copy from https://github.com/grpc/grpc/tree/master/test/core/iomgr/ios/CFStreamTests but instead point to the same shared test source.  If not, consider mark this as a copied ?

Reply:
It’s not possible to reference files in other directories in bazel.
This copy is temporary until bazel apple rules 2.0 release which added sanitizer support. 
https://github.com/bazelbuild/rules_apple/releases